components/main_menu/index.jsx

```diff
@@ -70,6 +71,10 @@ function mapStateToProps(state) {
     const joinableTeams = getJoinableTeamIds(state);
     const moreTeamsToJoin = joinableTeams && joinableTeams.length > 0;
     const rhsState = getRhsState(state);
+    const isCloud = getLicense(state).Cloud === 'true';
+    const subscription = state.entities.cloud.subscription;
```

Comment:
Can you remind me if we are loading this at initial load or if we need to wrap this in the `WithSubscription`?

Reply:
btw, it is already wrapped with the withSubscription hoc.

Reply:
Oh, didn't notice that it was already there. Thanks for the clarification!